packages/hardhat-zksync-deploy/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@matterlabs/hardhat-zksync-deploy",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Did anything change there?

Reply:
That was left from the previous publish after the #8 was merged. We should probably set up a GH Action to avoid this :thinking: